altimeter/aws/resource/iam/role.py

```diff
@@ -95,6 +103,8 @@ def list_from_aws(
                     role["PolicyAttachments"] = policies
                     resource_arn = role["Arn"]
                     roles[resource_arn] = role
+                    policies = get_embedded_role_policies(client, role_name)
+                    role["EmbeddedPolicies"] = policies
```

Comment:
Shouldn't the key name match line 45? In other words, shouldn't you use `EmbeddedPolicy` instead of `EmbeddedPolicies`?

Reply:
Absolutely, a last minute change on the name messed it up!! I'm changing it, and also I'm adding a test so it would not happen again.